LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2024 MongoDB, Inc.
```

Comment:
We apply the explicit copyright on a per-file basis. It's unclear to me whether we intentionally leave this line blank. AFAICT no other drivers explicitly apply a copyright. @blink1073 do you have insight?

Reply:
Correct.  This is the appendix, and is meant to be written as-is.  PyMongo and Java have the same appendix.

Reply:
Ah, I missed that this was the appendix.